keyboards/squiggle/keymaps/defaultminidox/keymap.c

```diff
@@ -0,0 +1,164 @@
+const uint16_t PROGMEM z_slash_combo[]       = {KC_Z, KC_SLSH, COMBO_END};
+const uint16_t PROGMEM x_comma_combo[]       = {KC_X, KC_COMM, COMBO_END};
+const uint16_t PROGMEM j_f_combo[]           = {KC_F, KC_J,    COMBO_END};
+
+combo_t key_combos[COMBO_COUNT] = {
+    // left hand combinations.
+    [Q_W]           = COMBO(q_w_combo, KC_TAB),
+    [W_E]           = COMBO(w_e_combo, KC_DQT),
+    [E_R]           = COMBO(e_r_combo, KC_UNDS),
+    [Z_X]           = COMBO(z_x_combo, KC_ENT),
+    [C_X]           = COMBO(x_c_combo, LCTL(KC_W)),
+    [C_V]           = COMBO(c_v_combo, KC_DELT),
+
+    // right hand combinations.
+    [P_O]           = COMBO(p_o_combo,       KC_BSPC),
+    [O_U]           = COMBO(o_u_combo,       KC_QUOT),
+    [U_I]           = COMBO(u_i_combo,       KC_MINS),
+    [SLASH_DOT]     = COMBO(slash_dot_combo, KC_PIPE),
+    [DOT_COMMA]     = COMBO(dot_comma_combo, KC_APP),
+    [COMMA_M]       = COMBO(comma_m_combo,   KC_DELT),
+
+    // both hand combinations.
+    [SLASH_Z]       = COMBO(z_slash_combo, KC_HOME),
+    [COMMA_X]       = COMBO(x_comma_combo, KC_END),
+    [J_F]           = COMBO(j_f_combo,     KC_ENT),
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[_BASE] = LAYOUT_minidox(
+    KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,       KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,
+    SH_A,   CT_S,   AL_D,   KC_F,   GU_G,       GU_H,   KC_J,   AL_K,   CT_L,   SH_SCLN,
+    KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,       KC_N,   KC_M,   KC_COMM,KC_DOT, KC_SLSH,
+                    LOWER,  KC_BSPC,SFT_ESC,    ALT_ENT,KC_SPC, RAISE
+),
+
+[_RAISE] = LAYOUT_minidox(
+    KC_EXLM,KC_AT,  KC_UP,  KC_LCBR,KC_RCBR,    KC_BSLS,KC_7,   KC_8,   KC_9,   KC_ASTR ,
+    KC_HASH,KC_LEFT,KC_DOWN,KC_RGHT,KC_DLR,     KC_EQL, KC_4,   KC_5,   KC_6,   KC_0 ,
+    KC_LBRC,KC_RBRC,KC_LPRN,KC_RPRN,KC_AMPR,    KC_GRV, KC_1,   KC_2,   KC_3,   KC_PLUS ,
+                    LOWER,  ADDDD,  _______,    ALT_ENT,RS_SPC, RAISE
+),
+[_LOWER] = LAYOUT_minidox(
+    KC_ESC, KC_QUES,KC_UNDS,KC_F1,  KC_F2,      KC_F3,  KC_F4,  KC_MINS,KC_SLSH,KC_BSPC ,
+    KC_LSFT,KC_TAB, KC_PGUP,KC_F5,  KC_F6,      KC_F7,  KC_F8,  KC_HOME,KC_LALT,KC_ENT  ,
+    KC_CLCK,KC_SLCK,KC_PGDN,KC_F9,  KC_F10,     KC_F11, KC_F12, KC_END, KC_INS, KC_SLSH ,
+                    LOWER,  _______,_______,    _______,ADDDD,  RAISE
+),
+[_ADJUST] = LAYOUT_minidox(
+    _______,EXPLR,  KC_UP,  PRVTAB, PRVWIN,     NXTWIN, NXTTAB, _______,_______,LCKGUI,
+    TSKMGR, KC_LEFT,KC_DOWN,KC_RGHT,UPTAB,      DNTAB,  KC_ENT, KC_LGUI,_______,CALDL,
+    _______,CLSGUI, _______,CONPST, RESET,      _______,_______,_______,_______,_______,
+                    _______,_______,_______,    _______,_______,_______
+),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+#ifdef CONSOLE_ENABLE
+    uprintf("KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
+#endif
+    return true;
+}
```

Comment:
```suggestion
```

Reply:
if I delete this, I couldn't get mod-tap working on the master's side. do you know why?

Reply:
Nope, that shouldn't be happening. All this is doing is printing to the console (if it's enabled).

Reply:
Oh, I enabled console feature in rules.mk, does that matter regarding the mod tap feature? or perhaps I should open different issue? 

ok, I will fix it as soon as I reach my desktop.